deployment/docker-compose.sh

```diff
@@ -89,7 +89,7 @@ addSudoIfNeeded() {
     }
 }
 
-cmdDocker="$(addSudoIfNeeded) env $(grep -v '^#' proxy.env | xargs) docker"
+cmdDocker="$(addSudoIfNeeded) env $(grep -v '^#' "$DIR/proxy.env" | xargs) docker"
```

Comment:
This change allows the `docker-compose.sh` script to be called from arbitrary folders and not only as `./docker-compose.sh`